.gitmodules

```diff
@@ -0,0 +1,10 @@
+[submodule "implementations/c/third-party/microchip/cryptoauthlib"]
```

Comment:
This, I suspect, is unsafe.
This build will break if the third-party commits a breaking change to their master.

We should move close a pinned commit of the repo to a gradle task.

Reply:
We're still in a heavy development phase. If they push an update I'd rather get it, have it break the build and understand what changed rather then lock us into one commit at the moment.